server/routes/produce.js

```diff
@@ -1,12 +1,12 @@
 const express = require('express')
 
-const db = require('../db/fruits')
+const db = require('../db/produce')
 
 const router = express.Router()
 
 router.get('/', async (req, res) => {
```

Comment:
I would recommend not returning all the fruit at once. This will make your api get slower and slower the more you add into it. I would look at adding a limit that you can pass into the api. This would also allow you to use the same api for multiple uses in the FE.